homeassistant/components/light/mqtt.py

```diff
@@ -44,14 +45,18 @@ def setup_platform(hass, config, add_devices_callback, discovery_info=None):
             'on': config.get('payload_on', DEFAULT_PAYLOAD_ON),
             'off': config.get('payload_off', DEFAULT_PAYLOAD_OFF)
         },
-        config.get('optimistic', DEFAULT_OPTIMISTIC))])
+        config.get('optimistic', DEFAULT_OPTIMISTIC),
+        {key: float(config.get(key + '_scale', DEFAULT_SCALE))
```

Comment:
I can understand scale for brightness but rgb ?  That's pretty much standardized to 255.